input/data/2014/2014A%26A...562A..40H/tev-000117.yaml

```diff
@@ -1,4 +1,23 @@
 source_id: 117
 reference_id: 2014A&A...562A..40H
 
-# TODO: add data
+data:
+  livetime: 120
+  significance: 8.2
+  excess: 117
+
+pos:
+  ra: {val: 18h18m4.8s, err: 3.1s, err_sys: }
```

Comment:
Section 2.3 states
> α = 18 18 4.8 ± 3.1stat

That means that the error on RA is 3 seconds (relative to an angle of 1 hour), not 3 arcseconds (which is seconds relative to an angle of 1 deg)
The correct way to put it is as string "0h0m0s" so that astropy Angle parses it correctly.